src/area/uploadArea.ts

```diff
@@ -362,10 +370,21 @@ class UploadArea {
 
     private isFileSizeValid(file: File): boolean {
         var maxFileSize = this.options.maxFileSize * 1024 * 1024; // max file size in bytes
-        if (file.size > maxFileSize) return false;
+        if (file.size > maxFileSize || file.size === 0) return false;
         return true;
     }
 
+    private isFileTypeInvalid(file: File): boolean {
+        if (file.name && (this.options.accept.trim() !== '*' || this.options.accept.trim() !== '*.*') && this.options.validateExtension) {
+            var acceptedExtensions = this.options.accept;
+            var fileExtension = file.name.substring(file.name.lastIndexOf('.'), file.name.length);
+			return acceptedExtensions.split(',').every((value) => {
```

Comment:
Replace the 'every' function with simple pure implementation with for cycle because of old IE browsers.